LCLSGeneral/LCLSGeneral/POUs/EPS/FB_EPS.TcPOU

```diff
@@ -26,6 +26,12 @@ END_VAR
 ]]></Declaration>
       <Implementation>
         <ST><![CDATA[nMask := SHL(nMask, nBits);
+// Checking the VAR_IN_OUT reference, leave the current method in case of invalid reference
+IF NOT __ISVALIDREF(eps) THEN
```

Comment:
How can a VAR_IN_OUT be an invalid reference? I'm not sure how to make a TwinCAT project that fails in this way.
The TwinCAT docs for this check indicate that it's only valid for REFERENCE TO declarations: https://infosys.beckhoff.com/english.php?content=../content/1033/tc3_plc_intro/2529165707.html&id=

Reply:
Nevermind, I see that these are in methods, so you can call them before setting the VAR_IN_OUT.